app/lib/gh-docs/index.ts

```diff
@@ -48,8 +48,9 @@ export function getRepoDocsMenu(ref: string, lang: string) {
   return getMenu(REPO, fixupRefName(ref), lang);
 }
 
-export function getRepoDoc(ref: string, slug: string) {
-  return getDoc(REPO, fixupRefName(ref), slug);
+export function getRepoDoc(ref: string, slug: string, dir?: string) {
+  let filepath = dir ? `${dir}/${slug}` : slug;
+  return getDoc(REPO, fixupRefName(ref), filepath);
```

Comment:
Nice! What do you think about removing the 3rd argument and just require passing in the full path to the slug? I think this simplifies the code you called out

```suggestion
export function getRepoDoc(ref: string, slug: string) {
  return getDoc(REPO, fixupRefName(ref), slug);
```